app/views/signup.py

```diff
@@ -23,3 +23,5 @@ def post(self):
                     return {"response":"user exists"},409
             return {"response": "Passwords do not match"}, 401
```

Comment:
its better you jsonify your responses than just returning a dictionary

Reply:
@Kasulejoseph 
Since am extending a Resource class, it jsonifies the dictionaries on return
otherwise if I jsonify again, will convert it into a strimg